jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/BuildDockerTask.java

```diff
@@ -105,37 +102,17 @@ public void buildDocker()
     TaskCommon.checkDeprecatedUsage(jibExtension, getLogger());
     TaskCommon.disableHttpLogging();
 
+    GradleProjectProperties projectProperties =
+        GradleProjectProperties.getForProject(getProject(), getLogger());
     try {
-      RawConfiguration gradleRawConfiguration = new GradleRawConfiguration(jibExtension);
-      GradleProjectProperties projectProperties =
-          GradleProjectProperties.getForProject(getProject(), getLogger());
-
-      PluginConfigurationProcessor pluginConfigurationProcessor =
-          PluginConfigurationProcessor.processCommonConfigurationForDockerDaemonImage(
-              gradleRawConfiguration,
+      PluginConfigurationProcessor.processCommonConfigurationForDockerDaemonImage(
```

Comment:
now that I see it, I think a name like `create....` or `get....` might make sense.

Reply:
Renamed the public methods to start with `createJibBuildRunner...` and also updated an out-of-date comment.